src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -1130,12 +2473,16 @@ module Tick : S = struct
                        + field
                            "0x0000000000000000000000000000000000000000000000000000000010000000"
                          * cell (var (Witness 6, Curr))
-                       - cell (var (Witness 1, Next)) ) ) ) )
+                       - cell (var (Witness 1, Next)) ) )
```

Comment:
Oops, what is this one about? Is it related to the if_feature inclusion or a missing piece of code independent of if_feature?